res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1712,6 +1940,10 @@ PioneerDDJSX.samplerLedsPlay = function(value, group, control) {
         sampleDeck = "[Sampler" + (samplerIndex + 1) + "]",
         padNum = PioneerDDJSX.samplerGroups[sampleDeck];
 
+    if (!engine.getValue(sampleDeck, "duration")) {
```

Comment:
Hmm, there might be a bug with the `track_loaded` Control. It works for me to show whether samplers are loaded.